src/autopas/containers/cellPairTraversals/CSlicedBasedTraversal.h

```diff
@@ -0,0 +1,235 @@
+      : CellPairTraversal<ParticleCell>(dims),
+        _overlap{},
+        _dimsPerLength{},
+        _interactionLength(interactionLength),
+        _cellLength(cellLength),
+        _overlapLongestAxis(0),
+        _sliceThickness{},
+        _dataLayoutConverter(pairwiseFunctor) {
+    init(dims);
+  }
+
+  /**
+   * Checks if the traversal is applicable to the current state of the domain.
+   * @return true iff the traversal can be applied.
+   */
+  bool isApplicable() const override {
+    return not(dataLayout == DataLayoutOption::cuda) and
+           this->_cellsPerDimension[_dimsPerLength[0]] >= _overlapLongestAxis;
+  }
+
+  /**
+   * Load Data Layouts required for this Traversal if cells have been set through setCellsToTraverse().
+   */
+  void initTraversal() override {
+    if (this->_cells) {
+      auto &cells = *(this->_cells);
+#ifdef AUTOPAS_OPENMP
+      /// @todo find a condition on when to use omp or when it is just overhead
+#pragma omp parallel for
+#endif
+      for (size_t i = 0; i < cells.size(); ++i) {
+        _dataLayoutConverter.loadDataLayout(cells[i]);
+      }
+    }
+    // split domain across its longest dimension
+
+    auto minSliceThickness = _overlapLongestAxis;
+    auto numSlices = static_cast<size_t>(this->_cellsPerDimension[_dimsPerLength[0]] / minSliceThickness);
+
+    _sliceThickness.clear();
+
+    // abort if domain is too small -> cleared _sliceThickness array indicates non applicability
+    if (numSlices < 1) return;
+
+    _sliceThickness.insert(_sliceThickness.begin(), numSlices, minSliceThickness);
+    auto rest = this->_cellsPerDimension[_dimsPerLength[0]] - _sliceThickness[0] * numSlices;
+    for (size_t i = 0; i < rest; ++i) ++_sliceThickness[i];
+    // decreases last _sliceThickness by _overlapLongestAxis to account for the way we handle base cells
+    _sliceThickness.back() -= _overlapLongestAxis;
+  }
+
+  /**
+   * Write Data to AoS if cells have been set through setCellsToTraverse().
+   */
+  void endTraversal() override {
+    if (this->_cells) {
+      auto &cells = *(this->_cells);
+#ifdef AUTOPAS_OPENMP
+      /// @todo find a condition on when to use omp or when it is just overhead
+#pragma omp parallel for
+#endif
+      for (size_t i = 0; i < cells.size(); ++i) {
+        _dataLayoutConverter.storeDataLayout(cells[i]);
+      }
+    }
+  }
+
+ protected:
+  /**
+   * Resets the cell structure of the traversal.
+   * @param dims
+   */
+  void init(const std::array<unsigned long, 3> &dims);
+
+  /**
+   * The main traversal of the colored sliced traversal.
+   *
+   * @copydetails C01BasedTraversal::c01Traversal()
+   *
+   * @tparam allCells Defines whether or not to iterate over all cells with the loop body given as argument. By default
+   * (allCells=false) it will not iterate over all cells and instead skip the last few cells, because they will be
+   * covered by the base step. If you plan to use the default base step of the traversal on this function, use
```

Comment:
This depends on the specific version for LinkedCells or VerletListsCells containers and is the same as for regular sliced.
ie c08 for LinkedCells and c18 for VerletListsCells if I'm not mistaken.

Reply:
Actually there is no default base step for a container. Sliced uses the c08 base step because it uses a c08 cell handler. VerletListsCells always processes everything in it's neighbor lists. Depending on whether they were built with Newton3 they contain only forward neighbors, resulting in a C18 base step or all neighbors resulting in C01.
So when can one set `allCells=false`? Actually always when doing a force calculation right? Because we never want to calculate interactions within the halo?

Reply:
As far as I can tell allCells=false isn't actually used anywhere. I can check if removing it breaks something :P